package.json

```diff
@@ -2,27 +2,519 @@
   "name": "to-do-list",
   "version": "1.0.0",
   "description": "",
-  "private": true,
+  "main": "webpack.config.js",
+  "dependencies": {
+    "accepts": "^1.3.8",
+    "acorn": "^8.8.2",
+    "acorn-import-assertions": "^1.8.0",
+    "acorn-jsx": "^5.3.2",
+    "ajv": "^6.12.6",
+    "ajv-formats": "^2.1.1",
+    "ajv-keywords": "^3.5.2",
+    "ansi-colors": "^4.1.3",
```

Comment:
- [ ] As required by the previous reviewer, Please set up the Webpack dev server correctly, currently, there are errors in the terminal when starting your application 👇🏻 

![Screenshot from 2023-02-01 20-59-19](https://user-images.githubusercontent.com/79058364/216141125-a34d005e-f155-4e5e-a166-d308d2c00ab7.png)


- Webpack is used to set up the project. Could you ensure the webpack dev server is set up and running without errors?